src/main/Root.js

```diff
@@ -50,6 +50,10 @@ class Root extends React.Component {
   }
 
   handleRangeChange(newRange: GenomeRange) {
+    // Do not propigate negative ranges
+    if (newRange.start < 0) {
```

Comment:
thanks for this change, @akmorrow13!

Do you think this would better behave in terms of the UI if we do `newRange.start = 0` instead of just doing nothing (i.e. `return`)?